regression/cbmc/byte_update15/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE broken-smt-backend
+CORE smt-backend
```

Comment:
Discussed offline: this one should just be "CORE", not restricted to "smt-backend"

Reply:
Changed as suggested.